### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by rishabhdeepsingh** [jmerle/competitive-companion#109]
Please support [https://m1.codeforces.com/](https://m1.codeforces.com/) (m2 and m3 as well).

**Comment by jmerle** [comment]
The Codeforces parser works with all of those subdomains since version 2.16.0.

**Comment by rishabhdeepsingh** [comment]
While parsing the whole contest on the contest page I am getting an error.
```
Competitive Companion could not determine which parser to parse this page with. Please right-click on the plus icon and select the parser to use via the "Parse with" context menu.
```

**Comment by jmerle** [comment]
Hmm, that's odd, I'll try to fix it during the next round.

**Comment by rishabhdeepsingh** [comment]
Thanks a lot.

**Comment by jmerle** [comment]
Fixed in 0f918f07acdd3d250291297f06f9b8905bbc08cc, will be released in a few hours.
